bin/masspay.py

```diff
@@ -193,6 +193,11 @@ def post_back_to_gratipay():
     except KeyError:
         gratipay_base_url = 'https://gratipay.com'
 
+    nmasspays = int(requests.get(gratipay_base_url + '/dashboard/nmasspays').text())
+    if nmasspays < 10:
```

Comment:
so this assumes there should be more than 10 teams to pay to? any way to override this without changing code?